source/jquery.flot.pie.js

```diff
@@ -119,6 +119,7 @@ More detail and specific examples can be found in the included HTML file.
             if (options.series.pie.show) {
                 if (options.grid.hoverable) {
                     eventHolder.unbind("mousemove").mousemove(onMouseMove);
+                    eventHolder.bind("mouseleave", onMouseMove);
```

Comment:
We should probably go ahead and provide the shutdown hook to this plugin, so we can unbind this event (see the hover plugin as an example)..